src/ng/directive/ngTranscludeLocals.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name ngTranscludeLocals
+ * @restrict A
+ *
+ * @description
+ * Directive that allows to define local variables for a given transclusion scope.
+ *
+ * It works with {@link ng.directive:ngTransclude ngTransclude} and it
+ * expects an object which define the local varibles for the transcluded template.
+ *
+ * @element ng-transclude
```

Comment:
I don't think this is accurate. It can be on any element (as long as it has `ngTransclude` on it).